ci/test.yml

```diff
@@ -419,5 +419,15 @@ connect-web mv2:
   after_script:
     - docker-compose down
     - docker network prune -f
+
+connect-web mv2:
+  extends: .connect-web mv2
   only:
     <<: *run_everything_rules
+  when: always
+
+connect-web mv2 manual:
```

Comment:
Adding the manual option to run this test comes in handy when we modify something that affects webextension we can run it before merging it and check all is working.